.eslint_dictionary.json

```diff
@@ -174,11 +180,13 @@
   "verifier",
   "versioned",
   "versioning",
+  "webp",
```

Comment:
This seems like it could be a typo of "web" so I'd rather just ignore spellcheck for a single line on this word if needed